SDL2_netConfig.cmake.in

```diff
@@ -12,3 +18,5 @@ endif()
 if(EXISTS "${CMAKE_CURRENT_LIST_DIR}/SDL2_net-static-targets.cmake")
     include("${CMAKE_CURRENT_LIST_DIR}/SDL2_net-static-targets.cmake")
 endif()
+
+check_required_components(SDL2_net)
```

Comment:
SDL2_net does not have components, so let's just do `set(SDL2_net_FOUND TRUE)`.
And if it had components you need to set `SDL2_net_FOUND` to a true-ish variable first: the function can only dsiable `SDL2_net_FOUND`, not enable it ([check its implementation](https://github.com/Kitware/CMake/blob/2eab65ca828084559e1019bd1263c5ab147bf135/Modules/CMakePackageConfigHelpers.cmake#L329-L337)).

I intent to add components to all SDL3 satellite libraries (`SDL3_xxx`, `SDL3_xxx-shared` and `SDL3_xxx-static`)

Reply:
> let's just do `set(SDL2_net_FOUND TRUE)`.

Is it necessary ? I tried loading a config without an explicit `set(SDL2_net_FOUND TRUE)` and without `check_required_components`, and `SDL_net_FOUND` was set to `1`.

Looking through [`find_package`'s source](https://gitlab.kitware.com/cmake/cmake/-/blob/master/Source/cmFindPackageCommand.cxx#L1480-1496), the command will only check if the package config explicitly set `<PackageName>_FOUND` to `FALSE` and will automatically [set the variable to 1](https://gitlab.kitware.com/cmake/cmake/-/blob/master/Source/cmFindPackageCommand.cxx#L1629) otherwise (or 0 if the package wasn't actually found).

Reply:
You're right.
The last lines [of the find_package documentation](https://cmake.org/cmake/help/latest/command/find_package.html) say you can set it to false, not that you are required to set it to true.